Makefile.am

```diff
@@ -58,7 +59,7 @@ AM_CPPFLAGS = \
 	$(NULL)
 
 # Utility library
-noinst_LTLIBRARIES += libeos-updater-util/libeos-updater-util-@EUU_API_VERSION@.la
+lib_LTLIBRARIES += libeos-updater-util/libeos-updater-util-@EUU_API_VERSION@.la
```

Comment:
These libraries will also need to be listed in `debian/*.install`, and you will need to add a new subpackage containing the GIR files. There’s some convention for naming the GIR subpackage which I can’t remember, but I’m sure you can cargo-cult it easily enough.